hieradata_aws/class/staging/jenkins.yaml

```diff
@@ -24,14 +25,7 @@ govuk_jenkins::job_builder::jobs:
   - govuk_jenkins::jobs::search_api_reindex_with_new_schema
   - govuk_jenkins::jobs::smokey
   - govuk_jenkins::jobs::smokey_deploy
-  - govuk_jenkins::jobs::data_sync
-  - govuk_jenkins::jobs::transition_import_dns
```

Comment:
Do these need to be ensured absent first?

Reply:
They can be deleted manually from the Jenkins UI.

Reply:
> Do these need to be ensured absent first?

They get deleted automatically by not being in the jenkins job list. I was likely more thorough than I needed to be in https://github.com/alphagov/govuk-puppet/pull/9319